services/web/server/src/simcore_service_webserver/projects/_nodes_handlers.py

```diff
@@ -168,6 +168,7 @@ async def create_node(request: web.Request) -> web.Response:
     return envelope_json_response(data, status_cls=web.HTTPCreated)
 
 
+# NOTE: Careful, this endpoint is actually "get_node_state," and it doesn't return a Node resource.
```

Comment:
TIP: add this note below the decorator or inside get_node function. This way it does not get accidentally pushed to the top by the auto-formatting tools